artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/browse.js

```diff
@@ -26,44 +26,55 @@ var ARTEMIS = (function(ARTEMIS) {
          displayName: 'Message ID',
          cellTemplate: '<div class="ngCellText"><a ng-click="openMessageDialog(row)">{{row.entity.messageID}}</a></div>',
          // for ng-grid
-         width: '10%'
-      }, {
-         field: 'userID',
-         displayName: 'User ID',
-         width: '10%'
+         width: '100px'
       }, {
          field: 'type',
          displayName: 'Type',
-         width: '10%'
+         cellTemplate: '<div class="ngCellText" title="{{row.entity.type}}">{{formatType(row.entity.type)}}</div>',
+         width: '90px'
       }, {
          field: 'durable',
          displayName: 'Durable',
-         width: '10%'
+         width: '70px'
       }, {
          field: 'priority',
          displayName: 'Priority',
-         width: '7%'
+         width: '70px'
       }, {
          field: 'timestamp',
          displayName: 'Timestamp',
-         width: '19%'
+         cellTemplate: '<div class="ngCellText" title="{{row.entity.timestamp}}">{{formatTimestamp(row.entity.timestamp)}}</div>',
+         width: '160px'
       }, {
          field: 'expiration',
          displayName: 'Expires',
-         width: '10%'
+         cellTemplate: '<div class="ngCellText" title="{{row.entity.expiration}}">{{formatExpires(row.entity.expiration)}}</div>',
+         width: '180px'
       }, {
          field: 'redelivered',
          displayName: 'Redelivered',
-         width: '10%'
+         width: '100px'
       }, {
          field: 'largeMessage',
          displayName: 'Large',
-         width: '10%'
+         width: '50px'
+      }, {
+         field: 'persistentSize',
+         displayName: 'Size',
+         cellTemplate: '<div class="ngCellText" title="{{row.entity.persistentSize}}">{{formatPersistentSize(row.entity.persistentSize)}}</div>',
+         width: '100px'
+      }, {
+         field: 'userID',
+         displayName: 'User ID',
+         cellTemplate: '<div class="ngCellText">{{row.entity.userID ? row.entity.userID : row.entity.StringProperties._AMQ_VALIDATED_USER}}</div>',
```

Comment:
@clebertsuconic I actually would prefer to show the _AMQ_VALIDATED_USER value by default and falling back to userID if it is not present. WDYT?

Reply:
thanks.. I didnt realize that.. thakns

Reply:
> @clebertsuconic I added another screenshot before the modifications to the JIRA issue.
thanks a lot, I didn't realize that before.

Reply:
> @clebertsuconic I actually would prefer to show the _AMQ_VALIDATED_USER value by default and falling back to userID if it is not present. WDYT?

would that be intuitive to users? shouldn't we keep it separated columns?
as long as users can understand the result without having to read any doc about it.

Reply:
> > @clebertsuconic I actually would prefer to show the _AMQ_VALIDATED_USER value by default and falling back to userID if it is not present. WDYT?
> 
> would that be intuitive to users? shouldn't we keep it separated columns?
> as long as users can understand the result without having to read any doc about it.

Sure, we could do that too.

Reply:
@clebertsuconic I added a separated validated user column and updated the screenshot in the JIRA issue.